Examples/Scripts/RunTICAPipeline_FullRun.sh

```diff
@@ -0,0 +1,273 @@
+#!/bin/bash
+
+set -eu
+
+# this is an example script to run the full tICA pipeline with new estimations on both group sICA and group tICA
+# beginning from the first step 'MIGP' to the second last step 'ComputeTICAFeatures'
+# since the step ClassifyTICA is not integrated yet, manual classification should be done after ComputeTICAFeatures,
+# then please rerun the pipeline from step CleanData to CleanData and specify the text file containing the component numbers to be removed by cleanup
```

Comment:
"from step CleanData to CleanData" ?

Reply:
"starting from step CleanData" would be better, I'll change it.  Specifying the stopping step is not needed in either of these examples, and was removed, but the comments weren't looked at as carefully.

Reply:
I have edited the comments at the top of both scripts to be more "user experience" and less "internal pipeline details".